docs/sections/user_guide/cli/mode_config.rst

```diff
@@ -169,4 +469,112 @@ TBD
 Examples
 ~~~~~~~~
 
-TBD
+The examples that follow use `JSON Schema <https://json-schema.org/understanding-json-schema/reference>`_ file ``schema.jsonschema`` with content
```

Comment:
We have an external link defined for JSON schema already, and can re-use it here either as a link to the main site as
```
:json-schema:`JSON Schema<>`
```
or to the specific page you're linking to here as
```
:json-schema:`JSON Schema<understanding-json-schema/reference>`
```

Reply:
Thanks, I had a feeling I was not doing that right